hm-post-repeat.php

```diff
@@ -144,7 +152,7 @@ function save_post_repeating_status() {
 	}
 
 	else {
-		update_post_meta( get_the_id(), 'hm-post-repeat', true );
+		update_post_meta( get_the_id(), 'hm-post-repeat', $_POST['hm-post-repeat'] );
```

Comment:
We'll want to sanitise this before we insert it into the database with something like `sanitize_text_field`